Gemfile

```diff
@@ -85,7 +85,7 @@ gem "inifile",                        "~>3.0",     :require => false
 gem "logging",                        "~>1.8",     :require => false  # Ziya depends on this
 gem "net_app_manageability",          ">=0.1.0",   :require => false
 gem "net-ping",                       "~>1.7.4",   :require => false
-gem "net-ssh",                        "~>2.9.2",   :require => false
+gem "net-ssh",                        "= 4.0.0.alpha4",   :require => false
```

Comment:
I do not want to change the gem version of net-ssh in darga from 2.9.x to 4.0-alpha.  First of all, all the changes in this PR are only in automate models.  We would need to validate `all` the places using it, which can be found via `git grep -i net::ssh`.
For Darga branch, the results of this command are:
```sh
app/controllers/host_controller.rb:      rescue Net::SSH::HostKeyMismatch => e   # Capture the Host key mismatch from the verify
app/models/host.rb:    rescue Net::SSH::AuthenticationFailed
app/models/host.rb:    rescue Net::SSH::HostKeyMismatch
app/models/host.rb:        rescue Net::SSH::HostKeyMismatch
db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/add_provider.rb:  Net::SSH.start($evm.root['deployment_master'], $evm.root['user'], :paranoid => false, :forward_agent => true,
db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/check_ssh.rb:    Net::SSH.start($evm.root['deployment_master'], $evm.root['user'], :paranoid => false, :forward_agent => true,
db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/create_ansible_inventory.rb:  Net::SSH.start($evm.root['deployment_master'], $evm.root['user'], :paranoid => false, :forward_agent => true,
db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/pre_deployment.rb:    Net::SSH.start($evm.root['deployment_master'], $evm.root['user'], :paranoid => false, :forward_agent => true,
gems/pending/util/MiqSshUtil.rb:require 'net/ssh/version'  unless defined?(Net::SSH::Version)
gems/pending/util/MiqSshUtil.rb:# puts "SSH Version: #{Net::SSH::Version::STRING}"
gems/pending/util/MiqSshUtil.rb:if Net::SSH::Version::MAJOR == 2
gems/pending/util/MiqSshUtil.rb:  raise "MiqSshUtil does not support version #{Net::SSH::Version::STRING} of Net::SSH"
gems/pending/util/MiqSshUtil.rb:  puts "SSH  Version: #{Net::SSH::Version::STRING}"
gems/pending/util/MiqSshUtilV2.rb:      Net::SSH.start(@host, @user, @options) do |ssh|
gems/pending/util/MiqSshUtilV2.rb:    rescue Net::SSH::HostKeyMismatch => e
```